core/include/drivers/rtc.h

```diff
@@ -12,14 +12,29 @@
 /* The RTC allows to set/get offset for correction */
 #define RTC_CORRECTION_FEATURE	BIT(0)
 
+struct rtc_subs;
+
+/*
+ * struct optee_rtc_time - Time capture
+ *
+ * @tm_year: Year of the capture
+ * @tm_mon: Month of the capture
+ * @tm_mday: Month day of the capture
+ * @tm_wday: Week day of the capture
+ * @tm_hour: Hour of the capture
+ * @tm_min: Minute of the capture
+ * @tm_sec: Second of the capture
+ * @tm_subs: Subsecond information of the capture. Implementation dependent
+ */
 struct optee_rtc_time {
 	uint32_t tm_year;
 	uint32_t tm_mon;
 	uint32_t tm_mday;
+	uint32_t tm_wday;
 	uint32_t tm_hour;
 	uint32_t tm_min;
 	uint32_t tm_sec;
-	uint32_t tm_wday;
+	uint32_t tm_subs;
 };
```

Comment:
By 'subsecond field' I meant milliseconds. I should have be more clear, sorry for that.
To me, this field cannot be implementation dependent as stated in the comments

Reply:
I agree. Having a millisecond (or even a microsecond) field in `struct optee_rtc_time` makes sense to me. RTC drivers will be free to fill them or not depending on the time granule the support.